src/porting.h

```diff
@@ -211,43 +211,58 @@ void initIrrlicht(irr::IrrlichtDevice * );
 	}
 
 #else // Posix
-
-	inline u32 getTimeS()
```

Comment:
I'll see in what places I can adjust the gap in documentation. This may just be a reason to get that wiki account.